src/test/fuzz/miniscript.cpp

```diff
@@ -769,6 +769,9 @@ NodeRef GenNode(F ConsumeNode, Type root_type, bool strict_valid = false) {
     std::vector<std::pair<Type, std::optional<NodeInfo>>> todo{{root_type, {}}};
     /** Predict the number of (static) script ops. */
     uint32_t ops{0};
+    /** Predict the total script size (every unexplored subnode is counted as one, as every leaf is
+     *  at least one script byte). */
+    uint32_t scriptsize{1};
```

Comment:
Maybe worth pointing to the explanatory comment in `Parse()`? I think the "borrowing" logic could be a little unclear to a reader with less context.